database_connect.php

```diff
@@ -0,0 +1,18 @@
+<?php
+$db = new PDO('mysql:host=db; dbname=lit_logger','root','password');
+$db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
I would just keep this bit in its own separate file because the db connection is the most reusable piece